core/src/backend/audio.rs

```diff
@@ -32,6 +32,7 @@ mod decoders {
     }
 }
 
+use ::swf::{CharacterId, SoundInfo};
```

Comment:
Why does this import start with `::`?

Reply:
I have no idea... huh.

I didn't know this was valid syntax? :D

Reply:
Fixed!